modules/dnn/test/test_layers_1d.cpp

```diff
@@ -480,4 +480,38 @@ INSTANTIATE_TEST_CASE_P(/*nothing*/, Layer_Scatter_Test, Combine(
 
 
 
+typedef testing::TestWithParam<tuple<std::vector<int>>> Layer_FullyConnected_Test;
+TEST_P(Layer_FullyConnected_Test, Accuracy)
```

Comment:
0d/1d or any other representative test name should be used.

Reply:
Done

Reply:
Probably commit not pushed to the branch